app/screens/settings/about/about.tsx

```diff
@@ -238,6 +267,22 @@ const About = ({componentId, config, license}: AboutProps) => {
                         {serverVersion}
                     </Text>
                 </View>
+                {license.IsLicensed === 'true' && loadMetric !== null && (
+                    <View style={styles.group}>
+                        <Text
+                            style={styles.leftHeading}
+                            testID='about.license_load_metric.title'
+                        >
+                            {intl.formatMessage({id: 'settings.about.license.load_metric.title', defaultMessage: 'Load Metric:'})}
+                        </Text>
+                        <Text
+                            style={styles.rightHeading}
+                            testID='about.license_load_metric.value'
+                        >
+                            {loadMetric}
+                        </Text>
+                    </View>
+                )}
```

Comment:
I'd love to see a test that cover this.

Reply:
Done! Though I'm not sure what to make of the overall Detox failures at the moment.